pulsar-client-cpp/run-unit-tests.sh

```diff
@@ -68,11 +68,31 @@ else
     RES=$?
 fi
 
+popd
+
+RES=0
```

Comment:
This will ignore the c++ test results.

Reply:
Good catch! I added that line since I was trying locally with the C++ tests commented out